content/1_General/Cpp_Command.mdx

```diff
@@ -234,9 +237,23 @@ If you type some integer and then press enter, then the program should produce o
 g++ name.cpp -o name && ./name
 ```
 
+Don't be afraid, the `./name` will only run if `g++ name.cpp -o name` finished successfully. Such is the magic of `&&`!
```

Comment:
> Don't be afraid,

??

Reply:
That is to say that you need not worry about `./name` running if the compilation wasn't successful.

Reply:
Maybe rephrase it into something slightly more formal to flow better with the rest of the module? (Maybe just say `&&` only executes the second command if the first command is successful, so `./name` will only run if the compilation finished successfully. or something?)

Reply:
Absolutely, will change it.